product_variant_available_in_pos/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,5 @@
+This module allows to have 'Available in the Point of Sale' flag at
+product level instead of product template.
+
+Please note this module is incompatible with `pos_product_template
```

Comment:
Put this in ROADMAP

Reply:
Thanks for the review!
That's seems right, done in https://github.com/OCA/product-variant/pull/102/commits/2528411d0a4974ecf7167b363cf40d9c46de302c